.github/workflows/build.yml

```diff
@@ -53,8 +53,7 @@ jobs:
         if: always() && github.repository == 'leanprover-community/mathlib' && github.event_name == 'push' && matrix.lean_version == 'leanprover-community/lean:3.5.1'
         run: |
           archive_name="$(git rev-parse HEAD).tar.gz"
-          tar czf "$archive_name" src
```

Comment:
Why did you remove the `tar`?

Reply:
Because I'm dumb and meant to remove archive_url instead.